### PR TITLE
feat(BackToTop): refactor with tokens

### LINKED ISSUE #6084: Back to Top tokens

### PR DESCRIPTION
**Pull request by wise-king-sullyman**
Closes #6084

[Figma design for back to top](https://www.figma.com/file/VMEX8Xg2nzhBX8rfBx53jp/Penta-Component-Library?type=design&node-id=3%3A17&mode=design&t=doZDRUaxE4vsn5dD-1)

Convenience link: https://patternfly-pr-6085.surge.sh/components/back-to-top

### CODE REVIEW

File: src/patternfly/components/BackToTop/back-to-top.scss [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
+  --#{$back-to-top}--Right: var(--pf-t--global--spacer--2xl);
+  --#{$back-to-top}--Bottom: var(--pf-t--global--spacer--lg);
+  --#{$back-to-top}--md--Bottom: var(--pf-t--global--spacer--2xl);
+  --#{$back-to-top}--c-button--FontSize: var(--pf-t--global--font--size--body--sm);
+  --#{$back-to-top}--c-button--BorderRadius: var(--pf-t--global--border--radius--large);
+  --#{$back-to-top}--c-button--PaddingTop: var(--pf-t--global--spacer--xs);
+  --#{$back-to-top}--c-button--PaddingRight: var(--pf-t--global--spacer--sm);
+  --#{$back-to-top}--c-button--PaddingBottom: var(--pf-t--global--spacer--xs);
+  --#{$back-to-top}--c-button--PaddingLeft: var(--pf-t--global--spacer--sm);
+  --#{$back-to-top}--c-button--BoxShadow: var(--pf-t--global--box-shadow--lg--bottom);
```
Comment (mcoker) [inline src/patternfly/components/BackToTop/back-to-top.scss:13]: Looks like the design calls for `/* light/global/shadow/small */`

File: src/patternfly/components/BackToTop/back-to-top.scss [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
-  --#{$back-to-top}--c-button--PaddingRight: var(--#{$pf-global}--spacer--sm);
-  --#{$back-to-top}--c-button--PaddingBottom: var(--#{$pf-global}--spacer--xs);
-  --#{$back-to-top}--c-button--PaddingLeft: var(--#{$pf-global}--spacer--sm);
-  --#{$back-to-top}--c-button--BoxShadow: var(--#{$pf-global}--BoxShadow--lg-bottom);
+:root {
+  --#{$back-to-top}--Right: var(--pf-t--global--spacer--2xl);
+  --#{$back-to-top}--Bottom: var(--pf-t--global--spacer--lg);
+  --#{$back-to-top}--md--Bottom: var(--pf-t--global--spacer--2xl);
+  --#{$back-to-top}--c-button--FontSize: var(--pf-t--global--font--size--body--sm);
+  --#{$back-to-top}--c-button--BorderRadius: var(--pf-t--global--border--radius--large);
```
Comment (mcoker) [inline src/patternfly/components/BackToTop/back-to-top.scss:8]: This uses the pill radius in design, but I think we can just remove this in favor of the pill radius on the button. I imagine the distinction between this button and the button component is just the font-size, padding, and box-shadow.

**Review by mcoker — CHANGES_REQUESTED** [review]
Looks good! Left a couple o' comments for review

**Comment by andrew-ronaldson** [comment]
Can we add `text-decoration: none` to remove the underline

**Review by mcoker — APPROVED** [review]
🤩

**Review by andrew-ronaldson — APPROVED** [review]
LGTM, thanks

**Pull request merged**